lisa/runners/lisa_runner.py

```diff
@@ -706,6 +706,25 @@ def _skip_test_results(
 
             test_result.set_status(TestStatus.SKIPPED, reasons)
 
+    def _ignore_features(self, nodes: List[schema.NodeSpace]) -> List[str]:
+        ignored_features: List[str] = []
```

Comment:
Use a set here, instead of convert at the end.

Reply:
update